### ISSUE TITLE
Jazzy seems to be not recognizing automatic manage signing in Xcode 8.3.2

### ISSUE THREAD

**Issue by alexiss03** [realm/jazzy#821]
Hi!

I run jazzy for my Xcode project that uses the _Xcode's automatic manage signing_.
But first, I tried running the project using `xcodebuild -x -workspace <project_workspace> -scheme <scheme>` that returned **build succeeded**, that caused my surprise when running jazzy produced an error.

I run with the command `jazzy`. The _.jazzy.yaml_ contains the following. 
```
xcodebuild_arguments:
    - "-workspace"
    - <project_workspace>
    - "-scheme"
    - <scheme>
```

The resulting log file seems incomplete with the last part not containing any errors. The start of the log file looks like this.
```
Build settings from command line:
    CODE_SIGN_IDENTITY =
    CODE_SIGNING_REQUIRED = NO
..................................
```

So, I tried running `jazzy` again but this time adding `-quiet` to _.jazzy.yaml_ in the hope finding the error.
```
xcodebuild_arguments:
    - "-workspace"
    - <project_workspace>
    - "-scheme"
    - <scheme>
    - quiet
```

After running `jazzy`, the resulting log file now contains.

_"Scheme Name" isn't code signed but requires entitlements. It is not possible to add entitlements to a binary without signing it._

I hope you can help me with this. 

Thanks,
Lexi

**Comment by johnfairh** [comment]
I don't know about signing or entitlements but a couple of things that might help people help you:
* what version of jazzy are you running?  If not latest, give it a go.
* is that your entire .jazzy.yml?
* could you please share the entire output from jazzy / any xcode log?

If this is a Swift project then I think sending xcodebuild `-quiet` is not going to help: the tools rely on getting the output from that command and I think this will make them break.

**Comment by alexiss03** [comment]
Hi!

I'm using the latest jazzy-0.8.2. As you suggested, I've updated my .jazzy.yaml file and removed ``-quiet``. Here is the updated whole content of my .jazzy.yaml file.

```
xcodebuild_arguments:
    - "-workspace"
    - "Pocket Note.xcworkspace"
    - "-scheme"
    - "Pocket Note"
```

Upon re-inspection, the log file now shows that the **BUILD SUCCEEDED** but jazzy still returns an error.

```
Could not parse compiler arguments from `xcodebuild` output.

Please confirm that `xcodebuild` is building a Swift module.
```

Also, I've uploaded the [log file](https://github.com/realm/jazzy/files/1024259/log.txt) as you requested. Hoping you can help me with this. Thank you!


Best,
Lexi

**Comment by johnfairh** [comment]
Thanks.  The problem is that sourcekitten is guessing "Pocket Note" as the name of your Swift module, but it is "Pocket_Note" according to the log file.

So I think you need to add `module_name: Pocket_Note` to .jazzy.yaml.

**Comment by alexiss03** [comment]
Hi!

I cannot confirm that this is the solution to the issue I've had earlier because I've changed the project name of my app to a single word, but positive that it is. Upon renaming my project, I tried running ``jazzy`` but still had the same error but when I added ``module: NewModuleName`` in .jazzy.yaml, it finally worked. By the way, jazzy does not recognize ``module_name``. Now it's working, I'm closing this issue. Again, thank you!

Best,
Lexi
